app/views/searches/show.html.erb

```diff
@@ -9,17 +9,7 @@
 <% if @gems %>
   <% @subtitle = t('.subtitle', :query => content_tag(:em, h(params[:query]))) %>
 
-  <% if @exact_match %>
-    <header class="gems__header">
-      <p class="gems__meter"><%= t '.exact_match' %></p>
```

Comment:
Is it ok to remove `exact_match` from translations as well?

Reply:
Thanks :+1: removed.